Frameworks/UIKit/UIActivityIndicatorView.mm

```diff
@@ -229,13 +242,12 @@ - (UIActivityIndicatorViewStyle)activityIndicatorViewStyle {
 */
 - (void)startAnimating {
     RunSynchronouslyOnMainThread(^{
-        if (_isAnimating) {
-            return;
+        if (!_isAnimating) {
+            _isAnimating = YES;
+            [self setHidden:NO];
```

Comment:
Yes, because by default iOS hides the indicator when it is not animating.

---
In reply to: [113078951](https://github.com/Microsoft/WinObjC/pull/2608#discussion_r113078951) [](ancestors = 113078951)